test/functional/p2p_compactblocks.py

```diff
@@ -785,6 +785,40 @@ def announce_cmpct_block(node, peer):
         stalling_peer.send_and_ping(msg)
         assert_equal(int(node.getbestblockhash(), 16), block.sha256)
 
+    def test_highbandwidth_mode_states_via_getpeerinfo(self):
+        # create new p2p connection for a fresh state w/o any prior sendcmpct messages sent
+        hb_test_node = self.nodes[0].add_p2p_connection(TestP2PConn(cmpct_version=2))
+
+        # Asserts that the RPC getpeerinfo boolean result fields "highbandwidth_{from,to}"
+        # for the last peer of a given node match the given parameters
+        def assert_highbandwidth_states(node, hb_from, hb_to):
```

Comment:
edb4ad4 suggestions
```diff
-        # Asserts that the RPC getpeerinfo boolean result fields "highbandwidth_{from,to}"
-        # for the last peer of a given node match the given parameters
-        def assert_highbandwidth_states(node, hb_from, hb_to):
+        # Assert the getpeerinfo `highbandwidth_to/highbandwith_from` boolean
+        # fields for the last peer of a given node match the given parameters.
+        def assert_highbandwidth_states_of_last_peer(node, hb_from, hb_to):
```

Reply:
Shortened the comment a bit and also reordered the parameters. Didn't want to change the function name to be even longer though (considering it's so close to where its used, possibly even closer after #19781 would be merged) everything should be clear.